src/components/POLineCard/POLineCard.test.js

```diff
@@ -0,0 +1,164 @@
+      'trial': true,
+      'expectedActivation': '2019-07-20T00:00:00.000+00:00',
+      'userLimit': 0,
+      'accessProvider': '14fb6608-cdf1-11e8-a8d5-f2801f1b9fd1',
+      'materialType': 'a7eb0130-7287-4485-b32c-b4b5814da0fa'
+    },
+    'fundDistribution': [
+      {
+        'code': 'USHIST',
+        'encumbrance': 'e1a607b4-2ed3-4bd9-9c1e-3726737d5425',
+        'fundId': '65032151-39a5-4cef-8810-5350eb316300',
+        'distributionType': 'percentage',
+        'value': 50
+      },
+      {
+        'code': 'EUROHIST',
+        'encumbrance': '6268d1e5-00c4-4ba7-86ff-c3eeb44886ea',
+        'fundId': 'e9285a1c-1dfc-4380-868c-e74073003f43',
+        'distributionType': 'percentage',
+        'value': 50
+      }
+    ],
+    'isPackage': true,
+    'locations': [
+      {
+        'locationId': 'f34d27c6-a8eb-461b-acd6-5dea81771e70',
+        'quantity': 2,
+        'quantityElectronic': 1,
+        'quantityPhysical': 1
+      },
+      {
+        'locationId': 'fcd64ce1-6995-48f0-840e-89ffa2288371',
+        'quantity': 1,
+        'quantityElectronic': 0,
+        'quantityPhysical': 1
+      },
+      {
+        'locationId': 'b241764c-1466-4e1d-a028-1a3684a5da87',
+        'quantity': 4,
+        'quantityElectronic': 0,
+        'quantityPhysical': 4
+      }
+    ],
+    'orderFormat': 'P/E Mix',
+    'paymentStatus': 'Pending',
+    'physical': {
+      'createInventory': 'Instance, Holding, Item',
+      'materialType': '5ee11d91-f7e8-481d-b079-65d708582ccc',
+      'materialSupplier': '70fb4e66-cdf1-11e8-a8d5-f2801f1b9fd1',
+      'receiptDue': '2018-08-19T00:00:00.000+00:00',
+      'volumes': [
+        'vol. 1'
+      ]
+    },
+    'poLineDescription': '',
+    'poLineNumber': '52590-1',
+    'purchaseOrderId': '0610be6d-0ddd-494b-b867-19f63d8b5d6d',
+    'receiptStatus': 'Pending',
+    'reportingCodes': '[]',
+    'requester': '',
+    'rush': false,
+    'selector': 'sgw',
+    'source': 'User',
+    'tags': {
+      'tagList': [
+        'membership'
+      ]
+    },
+    'titleOrPackage': 'Interesting Times',
+    'vendorDetail': {
+      'instructions': '',
+      'noteFromVendor': '',
+      'vendorAccount': '',
+      'referenceNumbers': []
+    },
+    'metadata': {
+      'createdDate': '2021-08-06T01:49:47.296+00:00',
+      'updatedDate': '2021-08-06T01:49:47.296+00:00'
+    },
+};
+
+let renderComponent;
+describe('POLineCard', () => {
+  beforeEach(() => {
+    renderComponent = renderWithIntl(
+      <Router>
+        <POLineCard
+          id="polines"
+          poLine={data}
+        />
+      </Router>,
+      translationsProperties
+    );
+  });
+ test('renders POLineCard component', () => {
+   const { getByTestId } = renderComponent;
+   expect(getByTestId('polines')).toBeInTheDocument();
+  });
+
+  test('renders a link with the poLineNumber', () => {
+    const { getByRole } = renderComponent;
+    expect(getByRole('link', { poLineNumber: '52590-1' })).toBeInTheDocument();
```

Comment:
@MonireRasouli Dont think `polLineNumber` is a valid option here. We should be using one from the list [here](https://testing-library.com/docs/queries/byrole/#api). 
```suggestion
    expect(getByRole('link', { name: 'PO line: 52590-1' })).toBeInTheDocument();
```